magicctapipe/scripts/lst1_magic/lst1_magic_stereo_reco.py

```diff
@@ -142,25 +153,36 @@ def stereo_reconstruction(input_file, output_dir, config, magic_only_analysis=Fa
     # Apply the event cuts
     logger.info(f"\nMAGIC-only analysis: {magic_only_analysis}")
 
+    LSTs_IDs = np.asarray(list(assigned_tel_ids.values())[0:4])
+
     if magic_only_analysis:
-        event_data.query("tel_id > 1", inplace=True)
+        event_data.query(f"tel_id > {LSTs_IDs.max()}", inplace=True) # Here we select only the events with the MAGIC tel_ids, i.e. above the maximum tel_id of the LSTs
```

Comment:
this relies on the sequence of LST and MAGIC ids

Reply:
Fixed (MAGIC IDs are the ones not in the LSTs_IDs list)

Reply:
ok, thanks, but the part of the comment " i.e. above the maximum tel_id of the LSTs" I think relates to the previous version and thus should be removed as well

Reply:
Fixed